reinforcement_learning/rl_unity_ray/rl_unity_ray.ipynb

```diff
@@ -544,9 +544,9 @@
  "metadata": {
   "anaconda-cloud": {},
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "conda_tensorflow_p36",
```

Comment:
is this a must have change?

Reply:
I believe so. When launching SM RL we tried to make sure the default kernel is consistent with the underlying example setting.

Reply:
`conda_tensorflow_p36` is preferable to `Python 3`, because it is the recommended kernel notebooks using TF1.x (for e.g., like this one). `Python 3` is not an available kernel on the notebook instance, and using it pushed the customer to select a kernel, which is bad UX imo.